readMe.md

```diff
@@ -0,0 +1,19 @@
+# Pokemon Tracker App
+Porfolio project to track prices, and ratings of pokemon cards. Created by Kobe Yang and Avery Tribbett, guided by Reid Williams.
```

Comment:
I'm definitely going to put this project in my portfolio, but I feel like we don't need to specify in the readme 😅

```
Comprehensive web-app used to track prices & ratings of Pokémon cards. 
Created by Kobe Yang and Avery Tribbett, guided by Reid Williams.
```

Looks more impressive 😎

Reply:
Yes, that is much better. Will do.